lib/mjsonwp/routes.js

```diff
@@ -9,7 +9,9 @@ const METHOD_MAP = {
     GET: {command: 'getStatus'}
   },
   '/wd/hub/session': {
-    POST: {command: 'createSession', payloadParams: {required: ['desiredCapabilities'], optional: ['requiredCapabilities', 'capabilities']}}
+    POST: {command: 'createSession',
+           payloadParams: {required: ['desiredCapabilities'],
+                           optional: ['requiredCapabilities', 'capabilities', 'alwaysMatch' , 'firstMatch']}}
```

Comment:
Looking at the [log](https://gist.github.com/TikhomirovSergey/28e7b0ffceacd480f44e1f0127edbfcb), that Mr Sergey kindly provided to us, I could say these properties are on the same level as desiredCapabilites. Anyway, I'd like to try first if this helps as a quick patch, so driver creation does not throw an error. Then we might think about parsing of the actual values.